scripts/experiments_tabular/GANomaly.jl

```diff
@@ -0,0 +1,115 @@
+using DrWatson
+@quickactivate
+using ArgParse
+using GenerativeAD
+using GenerativeAD.Models: anomaly_score, generalized_anomaly_score_gpu
+using BSON
+using StatsBase: fit!, predict, sample
+
+using Flux
+using MLDataPattern
+
+s = ArgParseSettings()
+@add_arg_table! s begin
+   "max_seed"
+		required = true
+		arg_type = Int
+		help = "seed"
+	"dataset"
+		required = true
+		arg_type = String
+		help = "dataset"
+end
+parsed_args = parse_args(ARGS, s)
+@unpack dataset, max_seed = parsed_args
+
+modelname ="GANomaly"
+
+function sample_params()
+    argnames = (
+        :hdim, 
+        :zdim, 
+        :nlayers, 
+        :activation, 
+        :lr, 
+        :batch_size, 
+        :iters, 
+        :check_every, 
+        :patience, 
+        :init_seed,
+    )
+    par_vec = (
+        2 .^(4:9),
+        2 .^(3:8),
+        3:4,
+        ["relu", "swish", "tanh"],
+        10f0 .^ (-4:-3),
+        2 .^ (5:7),
+        [10000],
+        [30],
+        [10],
```

Comment:
Is this enough? I have set patience to 200 in order to get out of certain local extremas.

Reply:
i am checking every 30 iterations with patience "10" => 30*10=300 iterations patience

Reply:
but i can change it to check_every = 1 and patience = 200 than it will be the same as yours

Reply:
ok. nevermind then